### PR TITLE
Rk cdl timeout

### PR DESCRIPTION
**Pull request by rakunkel-ucd**
timeout and error try/catch/continue
Bunyan logger

### CODE REVIEW

**Comment by rakunkel-ucd** [comment]
@qjhart - added Friday work to this PR

File: harvest/experts-client/package.json [hunk 1]
```diff
@@ -41,12 +41,13 @@
...
     "memory-level": "^1.0.0",
     "nanoid": "^4.0.2",
-    "node-fetch": "^3.3.1",
+    "node-fetch": "^3.3.2",
     "quadstore": "^11.0.8",
     "quadstore-comunica": "=3.0.0",
     "rdf-data-factory": "^1.1.0",
     "readable-stream": "^4.3.0",
     "string-to-stream": "^3.0.1",
+    "winston": "^3.11.0",
```
Comment (qjhart) [inline harvest/experts-client/package.json:50]: You can remove this right ?

**Review by qjhart — CHANGES_REQUESTED** [review]

**Comment by rakunkel-ucd** [comment]
Yes

Roger Kunkel
UC Davis Library
Online Strategy


From: Quinn Hart ***@***.***>
Date: Monday, November 27, 2023 at 4:41 PM
To: ucd-library/aggie-experts ***@***.***>
Cc: Roger Kunkel ***@***.***>, Author ***@***.***>
Subject: Re: [ucd-library/aggie-experts] Rk cdl timeout (PR #272)

@qjhart requested changes on this pull request.

________________________________

In harvest/experts-client/package.json<https://github.com/ucd-library/aggie-experts/pull/272#discussion_r1406942121>:

>      "quadstore": "^11.0.8",

     "quadstore-comunica": "=3.0.0",

     "rdf-data-factory": "^1.1.0",

     "readable-stream": "^4.3.0",

     "string-to-stream": "^3.0.1",

+    "winston": "^3.11.0",

You can remove this right ?

File: harvest/experts-client/package.json (hunk 1 above)
Comment (rakunkel-ucd) [reply harvest/experts-client/package.json]: Yes, removed.

**Review by qjhart — APPROVED** [review]

**Pull request merged**